internal/cli/atlas/deployments/podman/client.go

```diff
@@ -138,7 +138,7 @@ func (o *client) runPodman(ctx context.Context, arg ...string) ([]byte, error) {
 
 	cmd := exec.CommandContext(ctx, "podman", arg...)
 
-	output, err := cmd.CombinedOutput()
+	output, err := cmd.Output() // ignore stderr
```

Comment:
any reason for ignoring stderr as this might be useful information for debugging? (it's only printed if --debug flag is set)

Reply:
clarified offline